### DISCUSSION TITLE
spatie-laravel-translatable-plugin : make tenanteprofilepage translatable

### DISCUSSION THREAD

**Discussion by demobiel** [filamentphp/filament#8007] [Help]
I am having troubles in getting the translatable plugin to work on the edittenantprofile page...

for the plugin to work you have to have a translatable trait on both the resource and one one the page, but with the tenantprofilepage there is real resource of course...

**Comment by danharrin** [comment]
We require a resource to make things translatable, but tenants arent associated with a resource. It would need to be a completely new feature in the translatable package, which doesnt exist yet

**Comment by demobiel** [comment]
ok, so if I understand well, I should create a Translatable Trait for the EditTenantProfile (same as exists for EditRecord/CreateRecord in the plugin).
